### ISSUE TITLE
Use is_a() instead of creating an object  to check for its class

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/sql-parser#246]
It's [been suggested](https://github.com/phpmyadmin/sql-parser/pull/245#pullrequestreview-234555279) by @staabm that we could use "`is_a()` so we would not need to create a object just to check for its class".

This seems like a good idea to me.

**Comment by staabm** [comment]
I can work on that

**Comment by peter279k** [comment]
Just notice that this has the two approaches can check for the created class instance.

```php
$stdClass = new stdClass();

is_a($stdClass, stdClass::class); // true
$stdClass instanceof stdClass; // true
```

**Comment by peter279k** [comment]
Here is the `is_a` and `instanceof` comparison.

- `is_a` is a function, and it will be significantly slower (since it has all the overhead of executing a function call).
- instanceof is a language construct.  but the overall execution time is minimal in either method.

## References
- https://www.geeksforgeeks.org/what-is-the-difference-between-is_a-function-and-instanceof-in-php/

**Comment by williamdes** [comment]
Fixed by #290
